app/lib/LevelLoader.coffee

```diff
@@ -159,6 +160,8 @@ module.exports = class LevelLoader extends CocoClass
       url = "/db/level/#{@levelID}/session"
       if @team
         url += "?team=#{@team}"
+        if @level.isType('course-ladder') and @league and not @couseInstanceID
```

Comment:
```suggestion
        if @level.isType('course-ladder') and @league and not @courseInstanceID
```

Reply:
Also, is `@team` always set in these cases, or do we sometimes need to handle the `else if @courseID` branch below?

Reply:
@nwinter hey, i think `@team` will be always set since the bug comes from `play as red` or `play as blue` button.